tmt/steps/prepare/install.py

```diff
@@ -59,10 +80,57 @@ def go(self, guest):
                 self.info('copr', copr, 'green')
                 guest.execute(f'sudo dnf copr enable -y {copr}')
 
-        # Install packages
-        packages = self.get('package')
-        if packages:
-            self.info('package', fmf.utils.listed(packages, max=3), 'green')
+        # Check parameters, bail out if nothing to do
+        packages = self.get('package', [])
+        repo_packages = []
+        local_packages = []
+        directories = self.get('directory', [])
+        if not packages and not directories:
+            self.debug("No packages for installation found.", level=3)
+            return
+
+        # Detect local packages and directories
+        for package in packages:
+            if package.endswith('.rpm'):
+                local_packages.append(package)
```

Comment:
What if it's an URI? You can't simply copy that. It should be better detected. But I can address URIs in another PR.

Reply:
Yes, for now this is adding just local files support. Let's extend this in a separate pull request.